frontend/src/strings/crawl-workflows/infoText.ts

```diff
@@ -76,6 +76,17 @@ export const infoTextFor = {
   customBehavior: msg(
     `Enable custom page actions with behavior scripts. You can specify any publicly accessible URL or public Git repository.`,
   ),
+  failOnContentCheck: msg(
+    html`Fail the crawl if behaviors detect that the browser is not logged in on
+      specific pages. For more details see the
+      <a
+        href="https://crawler.docs.browsertrix.com/user-guide/behaviors/#fail-on-content-check"
+        class="text-blue-600 hover:text-blue-500"
+        target="_blank"
+        rel="noopener noreferrer nofollow"
+        >Fail On Content Check section of the crawler documentation</a
+      >.`,
+  ),
```

Comment:
Can we open the user guide here instead of taking users out of the application? The implementation can be similar to my suggestion this PR https://github.com/webrecorder/browsertrix/pull/2757#pullrequestreview-3049262885

The suggested implementation would also make it easier to translate the info text, since we wouldn't be including the entire `<a>` link markup in the translation string.

It could look something like:

```suggestion
${this.renderHelpTextCol(
  html`${infoTextFor["failOnContentCheck"]}
  ${this.renderUserGuideLink({
    hash: "fail-crawl-if-not-logged-in",
    content: msg("More details"),
  })}.`,
  false,
)}
```

Reply:
Hm, I do see the advantages of this. On the other hand this is something that may be updated more frequently, and is well documented in the crawler. I guess we could link to the user guide and then users could click through to the crawler docs for details?

Reply:
Yeah altogether I think this works well, so I went ahead and applied this suggestion.